factorial.py

```diff
@@ -0,0 +1,20 @@
+def factorial(n):
+    fact = 1
```

Comment:
The loop in the `factorial` function should include `n` in the range. It currently ends at `n-1` because `range(1, n)` excludes `n`. It should be `range(1, n + 1)`.